crates/teloxide/src/utils/command.rs

```diff
@@ -90,7 +90,7 @@ pub use teloxide_macros::BotCommands;
 ///  2. `#[command(prefix = "prefix")]`
 /// Change a prefix for all commands (the default is `/`).
 ///
-///  3. `#[command(description = "description")]`
+///  3. `#[command(description = "description")]` and `/// description`
```

Comment:
Why variant and enum attributes sections use different style?

Reply:
I did not understand

Reply:
Here you have

> 3. `#[command(description = "description")]` and `/// description`

while down below you have

> 3. `#[command(description = "description")]`

which then mentions documentation comment in the text.

I prefer the first style more.